.github/ISSUE_TEMPLATE/documentation_report.yml

```
@@ -46,8 +46,8 @@ body:
   attributes:
     label: Component Name
     description: >-
-      Write the short name of the rst file, module, plugin, task or
-      feature below, *use your best guess if unsure*.
+      Write the short name of the rst file, module, plugin, task or feature below,
```

```suggestion
      Write the short name of the file, module, plugin, task or feature below,
```
Docs could be RST, MD, or recently there is a PR where the submitter sent a .adoc file, and possible we might have others (.txt and what not). Instead of trying to list and keep it up to date, I suggest we take the "generic" road.